slsim/false_positive.py

```diff
@@ -0,0 +1,314 @@
+import numpy as np
+from lenstronomy.Cosmo.lens_cosmo import LensCosmo
+from slsim.ParamDistributions.los_config import LOSConfig
+from slsim.Util.param_util import ellipticity_slsim_to_lenstronomy
+from slsim.lens import theta_e_when_source_infinity
+
+
+class FalsePositive(object):
```

Comment:
there is a lot of duplicated code here with the Lens() class. Could you inherit the Lens() class and then only alter the definitions that really are different?

Reply:
Hi @Simon, Lens() class and FalsePositive() class has different inputs. Also, FalsePositive() class takes multiple sources while Lens class has only single source option. These difference create some conflicts while inheriting Lens class in FalsePositive class. My plan is to refactor Lens class first and then inherit in falsePositive class. But, this will make this PR very large. So, can you do this in next PR?

Reply:
@nkhadka21 (first, my tag is @sibirrer and not <at>simon (which notified someone totally unrelated)). How urgently does this PR need to be merged? The reason I am asking is also that we could refactor in parallel the Lens() class in a different PR

Reply:
Thank you @sibirrer for correcting this! I think it not very urgent to merge this PR. I agree that we can refactor Lens() class first and then update this PR.